notebooks/official/tabnet/sdk2_remote_tabnet_training.ipynb

```diff
@@ -0,0 +1,693 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    # @title Copyright &amp; License (click to expand)
</pre><p>Remove this line.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
done